regression/systemc/Array4/test.desc

```diff
@@ -1,8 +1,10 @@
-CORE
+KNOWNBUG
 main.cpp
 -DNO_IO -DNO_STRING
 ^EXIT=0$
 ^SIGNAL=0$
 ^VERIFICATION SUCCESSFUL$
 --
 ^warning: ignoring
+--
+warning: ignoring byte_extract_little_endian
```

Comment:
Do you mean this to be a comment?

Reply:
Yeah... along with the previous line that does seem a little bizaar.